changelog.json

```diff
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "2.4.4-beta2": [
+      "[Fixed] Enable tags feature flags for beta - #9682",
```

Comment:
This is internal stuff right? So I think we can just nix this -- what do you think? otherwise we should just say [Fixed] show unpushed... so basically what we had intended to do but it's "fixed" from the user's perspective

Reply:
```suggestion
      "[Fixed] Display tags in commit list and show tag creation option - #9682",
```